01_introduction_to_algorithms/ruby/01_binary_search.rb

```diff
@@ -1,26 +0,0 @@
-def binary_search(list, item)
```

Comment:
@Salvi6God I really like this idea, but I also want to preserve the simple structure of the ruby directory. For people who don't care about tests, I want them to be able to see a single file that contains the code for binary search. Can you please leave this file as is, and add your changes to `01_introduction_to_algorithms/ruby-with-tests/` ? That way the examples with tests will have their own home, and others will be encouraged to add more examples with tests.